client/environment.go

```diff
@@ -97,11 +97,24 @@ func (Environment) getEndpoint() string {
 }
 
 func (client *ApiClient) Environments() ([]Environment, error) {
-	return getAll(client, nil)
+	organizationId, err := client.organizationId()
+	if err != nil {
+		return nil, err
+	}
+
+	return getAll(client, map[string]string{
+		"organizationId": organizationId,
+		"isActive":       "true",
+		"onlyMy":         "false",
```

Comment:
@sagilaufer1992 
The idea was to keep the behavior of the old API call.

When I review the call in the UX, these fields are passed.

I don't mind removing them, but I will still have to pass them somewhere (as they are needed). I could do it in resources/data code instead. Not sure what's better.

Reply:
you may keep them, but not as constants. onlyMy may be true or false even in the app. this way really limits the usage and is NOT backward compatible, as before the query was less restrictive in regards to those options.
resource or data, which is better, hard for me to tell... @yaronya care to help us with that?

Reply:
We should avoid introducing any breaking change and make sure the environments list returned is the same as before

Reply:
I'm not in the details of this issue so please share some more info. 
Which new mandatory fields fail the API call now?

Reply:
@yaronya - sorry.
The /environments call no longer works.
For it to be equivalent I need path new params such as 
```
"isActive":  "true",
"onlyMy":    "false",
```

@sagilaufer1992 is thinking it might be better to have these fields be passed explicitly when calling environments().
(E.g. `environments(isActive boolean, onlyMy boolean)`).
I was thinking it's better to path them implicitly (to maintain compatibility with the old `environments()` call). (This is the way I implemented it in this PR).

I guess we're in a disagreement and we need someone to help us decide what makes more sense (:

Reply:
So I don't think it's that crucial, but if I gotta choose - I'd prefer to have it implicitly since there isn't great benefit IMO in doing it otherwise.

Another question tho - previously, the provider used to fetch also non-active environments? Can we move forward without stating `isActive`? I'd just want to avoid the scenario of users getting different list now.

Reply:
oh! I actually thought it was the other way around.
Okay. Then I think we can actually remove these two fields and that should resolve our discussion.

Thanks!